catalyst/utils/metrics/ndcg.py

```diff
@@ -0,0 +1,72 @@
+"""
+Cumulative Gain metrics:
+    * :func:`dcg`
+    * :func:`ndcg`
+"""
+import torch
+
+
+def dcg(
```

Comment:
could you please add tests for new metrics?

Reply:
and add them to `catalyst/docs/utils.rst`?

Reply:
btw, what do you think about adding `(N)DcgCallback`?

Reply:
could you please also add docs for the new metrics and callback? 
I should looks like https://github.com/catalyst-team/catalyst/blob/master/catalyst/dl/callbacks/metrics/iou.py#L12